caveman-test.asd

```diff
@@ -6,24 +6,17 @@
   Caveman is freely distributable under the LLGPL License.
 |#
 
-(in-package :cl-user)
-(defpackage caveman-test-asd
-  (:use :cl :asdf))
-(in-package :caveman-test-asd)
-
-(defsystem caveman-test
-  :depends-on (:caveman
-               :cl-test-more
-               :uiop
-               :dexador
-               :usocket)
+(defsystem "caveman-test"
+  :depends-on ("caveman"
+               "cl-test-more"
+               "uiop"
+               "dexador"
+               "usocket")
```

Comment:
What's the advantage of using strings over a symbol or an atom?

Reply:
It's canonical.